public/app/percona/settings/components/Platform/Connect/Connect.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory(({ breakpoints, colors, spacing, typography }: GrafanaTheme) => ({
```

Comment:
I think `stylesFactory` is not needed here. Same in `Connected.styles`

Reply:
Thank you!